src/applications/edu-benefits/0993/containers/SubmitController.jsx

```diff
@@ -27,7 +27,11 @@ class SubmitController extends React.Component {
   }
 
   goBack = () => {
-    this.props.router.goBack();
```

Comment:
We could add a property to the manifest to accomplish this. It's weird case where our `rootUrl` doesn't redirect to the intro page.

Reply:
Yes, good idea, and that could also potentially be used to add the static page to the ignorelist, if an additional production flag for the static page is provided. Will create a ticket for this.